### ISSUE TITLE
minimal version (or instructions to make one)

### ISSUE THREAD

**Issue by My1** [insites/cookieconsent#256]
is there a way to strip unneeded stuff from this thing?

while some features are nice, I think it would be nice not to have 20KB of (already minimized!) Javascript and 4KB of CSS for stuff that isnt really needed by one, and a way to know what to strip if it isnt needed would be nice.

at least a relatively simple
~~~
onload = function() {
  if(not_dismissed) {
    show_it();
  }
}
dismiss.onclick = function {
  setcookie(dismissed=yes);
  hide_it();
}
~~~
shouldnt need that amount of script especially minimized.

so basically having a list of what feature needs what functions etc, for a simple removal would be awesome.

**Comment by alexmorleyfinch** [comment]
If size is really important, I'd recommend rolling your own solution. We need this config. There is no way we can disable your cookies for you, you have to hook that code in.

**Comment by garygreen** [comment]
Agreed. This library I have to say is pretty ridiculous in it's size. It's almost the size of Vue and nearing the size of jQuery all for a cookie consent! We currently have this code split so it's only loaded on demand. And as we use Cloudflare we already have country detection so don't need anything fancy inside the library in that aspect.

I wish this library was more modular and minimal in it's implementation.
